<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataLog_Files</class>
 <widget class="QWidget" name="DataLog_Files">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="file_menu_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="saving_file_label">
        <property name="maximumSize">
         <size>
          <width>293</width>
          <height>91</height>
         </size>
        </property>
        <property name="text">
         <string>Path/File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="saving_file_line"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="validation_button">
     <property name="text">
      <string>Validate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
